synapse/rest/client/capabilities.py

```diff
@@ -73,6 +73,9 @@ async def on_GET(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
                 "enabled": self.config.registration.enable_3pid_changes
             }
 
+        if self.config.experimental.msc3440_enabled:
+            response["capabilities"]["io.element.thread"] = {"enabled": True}
```

Comment:
I think this should maybe be `unstable.*`. will ask the SCT for opinions

Reply:
I think in the past we've just namespaced them (i.e. not using `m.*`), but easy enough to change as long as clients know.

Reply:
seems I was being a bit of a crank.